guide/inheritance-wallet/backup.md

```diff
@@ -0,0 +1,279 @@
+   retina = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-scenarios@2x.png"
+   mobile = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-scenarios-mobile.png"
+   mobileRetina = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-scenarios-mobile@2x.png"
+   alt-text = "A diagram showing the sequence of recovery scenarios."
+   width = 1304
+   height = 480
+%}
+
+To achieve this goal, their backup strategy accounts for the following three scenarios: 
+
+1. **Alice and Bob need to recover the wallet themselves**: They might simply need to restore the wallet because one (or both) of them lost access to their phone or computer. This route also accounts for one of the most most likely scenarios in which one spouse outlives the other 
+1. **A self-sovereign inheritance route for Christina and David**: The children should be able to use their inheritance keys and the backup material without any outside help
+1. **An assisted inheritance route** that provides e a fallback in case the self-sovereign inheritance fails. It is designed to help Christina and David access the funds through regular inheritance proceedings. 
+
+As we have covered on the wallet creation page, both of the inheritance routes for Christina and David are available [after six months]({{ '/guide/inheritance-wallet/wallet-creation/#creating-the-inheritance-key-set' | relative_url }}) of wallet inactivity. 
+
+## Backup material distribution
+
+The diagram below provides a high-level overview of how Alice and Bob distribute the backup material to achieve this goal. It supports the recovery scenarios outlined above, while maintaining financial privacy for Alice and Bob during their lifetimes.
+
+{% include picture.html
+   image = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution.png"
+   retina = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution@2x.png"
+   modalImage = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution@2x.png"
+   mobile = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-mobile.png"
+   mobileRetina = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-mobile@2x.png"
+   modalImageMobile = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-mobile@2x.png"
+   alt-text = "A schematic illustration showing how the backup material is distributed between the parents, their children and the lawyer."
+   width = 1600
+   height = 800
+   modalWidth = 3324
+   modalHeight = 1850
+   modalWidthMobile = 1156
+   modalHeightMobile = 2936
+%}
+
+
+### Password Manager
+
+##### Alice and Bob's personal vaults
+
+Alice and Bob each store an encrypted version of the recovery kit in their own personal vaults of the password manager they use. Encrypting the recovery kit protects it from a potential security breach on the side of the password manager. 
```

Comment:
Personal vault and password manager seem almost the same thing here or would there be a difference?

Reply:
Yes, there is a difference:  The password manager is the overall application, while a vault is a dedicated space inside of this application, where users can store passwords/information. In our case, Alice and Bob each have their own space (vault) that only they have access to.  

In addition, they created a shared family vault with shared access for all family members. So everybody will be able to access the material in the family vault, but not the material in Alice's and Bob's personal vaults.